tensorflow/core/common_runtime/mkl_cpu_allocator.h

```diff
@@ -139,27 +142,27 @@ class MklSmallSizeAllocator : public VisitableAllocator {
                                     static_cast<size_t>(stats_.max_alloc_size));
   }
 
-  /// Decrement statistics for the allocator handling small allocations.
-  inline void DecrementStats(size_t dealloc_size) {
+  // Decrement statistics for the allocator handling small allocations.
+  inline void DecrementStats(size_t dealloc_size) GUARDED_BY(mutex_) {
```

Comment:
Same as the comment above. Please replace `GUARDED_BY(mutex_)` with `EXCLUSIVE_LOCKS_REQUIRED(mutex_)`.